examples/labs/alex_controller/main.py

```diff
@@ -0,0 +1,16 @@
+from pitop import DriveController, PanTiltController, Camera
+from pitop.labs import WebServer, AlexControllerBlueprint
+
+drive = DriveController()
+pan_tilt = PanTiltController()
+camera = Camera()
+
+alex_controller = WebServer(
+    blueprint=AlexControllerBlueprint(
+        camera=camera,
+        drive=drive,
+        pan_tilt=pan_tilt
+    )
```

Comment:
Since this is for an Alex robot I think we should probably require people to do this the robot way:
```
robot = Pitop()

robot.add_component(drive_controller)
robot.add_component(pan_tilt)
robot.add_component(camera)

alex_controller = WebServer(
    blueprint=AlexControllerBlueprint(robot=robot)
)
```

Reply:
and also maybe it doesn't have to be for alex, i think we were calling it robot controller before this PR, so maybe RobotControllerBlueprint for now until we specialise more

Reply:
On slack we settled on adding the AlexWebController class to make this pattern more concise/convenient.